website/docs/d/interface_config.html.markdown

```diff
@@ -44,11 +44,15 @@ data "aci_interface_config" "access_port_config_1003" {
 
 ## Attribute Reference ##
 * `id` - The Attribute ID set to the Dn of the Port Configuration.
-* `role` - (Optional) The Role of the Port Configuration object. Type: String.
-* `policy_group` - (Optional) The Distinguished Name of the Policy Group being associated with the Port Configuration object. Type: String.
-* `breakout` - (Optional) The Breakout Map of the Port Configuration object. Type: String.
-* `admin_state` - (Optional) The Admin State of the Port Configuration object. Type: String.
-* `pc_member` - (Optional) The Distinguished Name of the Port Channel Member being associated with the Port Configuration object. Type: String.
-* `description` - (Optional) The Description of the Port Configuration object. Type: String.
-* `annotation` - (Optional) The Annotation of the Port Configuration object. Type: String.
-* `name_alias` - (Optional) The Name Alias of the Port Configuration object. Type: String.
+* `role` - (Read-Only) The Role of the Port Configuration object. Type: String.
+* `policy_group` - (Read-Only) The Distinguished Name of the Policy Group being associated with the Port Configuration object. Type: String.
+* `breakout` - (Read-Only) The Breakout Map of the Port Configuration object. Type: String.
+* `admin_state` - (Read-Only) The Admin State of the Port Configuration object. Type: String.
+* `pc_member` - (Read-Only) The Distinguished Name of the Port Channel Member being associated with the Port Configuration object. Type: String.
+* `description` - (Read-Only) The Description of the Port Configuration object. Type: String.
+* `annotation` - (Read-Only) The Annotation of the Port Configuration object. Type: String.
+* `name_alias` - (Read-Only) The Name Alias of the Port Configuration object. Type: String.
```

Comment:
You should remove the `Optional` flag from the data source when you say `Read-Only`. Otherwise, the Provider will allow to keep the values in the configuration file.

Reply:
There's no (Optional) here?

Reply:
I think he is talking about the older attributes in the data source schema. They still have optional.

Reply:
I removed them. Thanks